### PR TITLE
2018 November Planning Session Notes, Update Lead Roles

### PR DESCRIPTION
**Pull request by dasanchez**

### CODE REVIEW

**Comment by Shrinks99** [comment]
https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md Should be updated accordingly.

- People being around for one cycle before assigned lead roles
- Updated lead list

Anything else I'm missing?  If we want to do that in a separate PR that's fine and this looks good.  If you want those bundled with this one I'd be cool with that.  Either way will approve once that's decided upon.

**Comment by dasanchez** [comment]
Thanks @Shrinks99! It makes sense to me to revise the coordination structure doc at this time.

**Review by Shrinks99 — APPROVED** [review]
Looks good to me!

**Review by benhylau — COMMENTED** [review]
Minor

File: meeting_notes/20181122_planning-meeting-notes.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  - look into storage options
+  - Issue: https://github.com/tomeshnet/p2p-internet-workshop/issues/56
+  - PR: https://github.com/tomeshnet/documents/pull/99
+- benhylau can run a session on how to facilitate workshops in December
+
+Node / tech development
+
+- Adding SSB (patchfoo) to the prototype stack
+- See v0.4 roadmap: Yggdrasil + Install Profiles in prototype
+- Contact at Alpha Networks if anyone's interested in buying from them
```
Comment (benhylau) [inline meeting_notes/20181122_planning-meeting-notes.md]:
```suggestion
- Contact at [Alfa Network](https://www.alfa.com.tw) if anyone's interested in buying from them
```

**Review by benhylau — APPROVED** [review]

**Review by darkdrgn2k — APPROVED** [review]

**Pull request merged**